app/js/event/event.filter.js

```diff
@@ -0,0 +1,59 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('upcomingStops.event')
+        .factory('eventFilter', eventFilter);
```

Comment:
Ok, this is a factory, sorry I misread before that this would be a filter. Factory is probably better. Though factory should have a single function that returns function(s).